metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/AggregationHelper.java

```diff
@@ -140,4 +156,25 @@ public static Optional<MemoryMetric> getAggregatedMemory(List<Execution> executi
         }
         return Optional.empty();
     }
+
+    /**
+     * Aggregate Validation metrics from the list of validation executions by performing a logical AND.
```

Comment:
Ditto, I think this is the section which will need a few more additional statistics IIRC